.github/workflows/ci.yml

```diff
@@ -71,6 +73,33 @@ jobs:
           fetch-depth: 0
           ref: ${{ github.event.pull_request.head.ref }}
           repository: ${{ github.event.pull_request.head.repo.full_name }}
+          submodules: true
+          token: ${{ secrets.PAT_REVIEW }}
+
+      - name: Use cassettes previously sent
```

Comment:
only use the cassettes if you're working on a pull request and there is no conflict with master